src/main/kotlin/no/digdir/fdk/searchservice/model/CommonDataClasses.kt

```diff
@@ -71,3 +72,30 @@ data class Definition(
   val sources: Sources?,
   val sourceRelationship: String?,
 )
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+class HarvestMetaData(
+  val firstHarvested: String?,
+  val changed: List<String>?,
+)
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class Catalog(
+  val description: LocalizedStrings?,
+  val id: String?,
+  val publisher: Publisher?,
+  val title: LocalizedStrings?,
+  val uri: String?,
+)
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class Organization(
+  val title: LocalizedStrings?,
```

Comment:
Og bruk denne i stedet for `Publisher`
```suggestion
  val name: String?,
  val prefLabel: LocalizedStrings?,
```